base/formats.go

```diff
@@ -197,3 +199,29 @@ func TrimBase64Mime(b64 string) string {
 	splitB64 := strings.Split(b64, ",")
 	return splitB64[len(splitB64)-1]
 }
+
+// return the extension of the file from the base64 string, in the "jpeg" , "png" format, check with magic number
```

Comment:
We probably want to add the test code for these functions.

Reply:
added testcase for these two functions